im.bernard.Memorado.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id" : "im.bernard.Memorado",
```

Comment:
How to verify that domain belongs to the project here? The GitHub repo doesn't have a link to it, the page has nothing in it.

Reply:
Added a URL to the Github page for now, I'll make a little website for it later.

Reply:
Update: There's now a website here http://bernard.im/memorado

Reply:
Thanks